src/pharmpy/tools/mfl/parse.py

```diff
@@ -309,6 +315,38 @@ def replace(self, **kwargs):
             metabolite=metabolite,
         )
 
+    def replace_features(self, mfl_str):
+        key_dict = {
+            Absorption: 'absorption',
+            Elimination: 'elimination',
+            Transits: 'transits',
+            Peripherals: 'peripherals',
+            LagTime: 'lagtime',
+            Covariate: 'covariate',
+            DirectEffect: 'direct_effect',
+            EffectComp: 'effect_comp',
+            IndirectEffect: 'indirect_effect',
+            Metabolite: 'metabolite',
+        }
+        mfl_list = _parse(mfl_str)
+        kwargs = {}
+        for statement in mfl_list:
+            key = key_dict[type(statement)]
+            if type(statement) in (Transits, Peripherals, Covariate, IndirectEffect):
+                no_of_modes = len(statement.counts)
+                value = (statement,)
+            else:
+                no_of_modes = len(statement.modes)
+                value = statement
+            if no_of_modes > 1:
```

Comment:
I am not entirely sure I follow why you would not be allowed to replace an attribute with a feature with more than one "mode". For instance `ABSORPTION([FO,ZO])`. Is there a specific reason as to why this should not be allowed? As it is supported in the class.

Reply:
Fair point, originally I came from the side of the modelbuilder where it's always one feature at the time, but I don't think it needs to be restricted to that. Will change!